spec/models/queue_item_spec.rb

```diff
@@ -43,4 +47,33 @@
       expect(queue_item.category).to eq video.category
     end
   end
+
+  describe "#update_rating" do
+    let(:user) { Fabricate(:user) }
+    let(:video) { Fabricate(:video) }
+    let(:queue_item) { Fabricate(:queue_item, user: user, video: video) }
+
+    context "review is not present" do
+      it "creates a new review" do
+        queue_item.rating = 5
+        expect(Review.count).to eq 1
```

Comment:
You can write this as `expect { queue_item.rating = 5 }.to change { Review.count }`.